pages/common/ack.md

```diff
@@ -19,6 +19,14 @@
 
 `ack -cl {{foo}}`
 
+- Search file "my_file" for string "foo bar":
+
+`ack bar "{{foo bar}}" {{my_file}}`
```

Comment:
```suggestion
`ack bar "{{bar}}" {{path/to/file}}`
```

Reply:
I wanted to show a multi-word string in this example.

Reply:
> I wanted to show a multi-word string in this example.

You want to use `ack` to seach `foo` or `bar` patterns in the file? If so I think you can do this

```suggestion
`ack "{{foo | bar}}" {{path/to/file}}`
```

Reply:
I think what @cam8001 means is that they wanted to show specifically an example with a space in it here.